Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -133,6 +138,16 @@ public struct CustomerCenterConfigData {
                     return "Update"
                 case .updateWarningIgnore:
                     return "Continue"
+                case .platformMismatch:
+                    return "Platform mismatch"
+                case .pleaseContactSupportToManage:
+                    return "Please contact support to manage your subscription."
```

Comment:
Maybe not for this PR, but would be great if there were a link/button that would take them directly to contacting support from this page.

Reply:
That button is probably something you can configure from the JSON :)